api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -191,6 +191,42 @@ type VolumeReplicationGroupSpec struct {
 	KubeObjectProtection *KubeObjectProtectionSpec `json:"kubeObjectProtection,omitempty"`
 }
 
+// StorageIdentifiers carries various identifiers that help correlate the identify of a storage instance
+// that is backing a PVC across kubernetes clusters.
+type StorageIdentifiers struct {
+	// CSIProvisioners contains the provisioner name of the CSI driver used to provision this
+	// PVC (extracted from the storageClass that was used for provisioning)
+	//+optional
+	CSIProvisioner string `json:"csiProvisioner,omitempty"`
+
+	// StorageID contains the globally unique storage identifier, as reported by the storage backend
+	// on the StorageClass as the value for the label "ramendr.openshift.io/storageID", that identifies
+	// the storage backend that was used to provision the volume. It is used to label different StorageClasses
+	// across different kubernetes clusters, that potentially share the same storage backend
+	//+optional
+	StorageID string `json:"storageID,omitempty"`
```

Comment:
We talked about these two, StorageID and ReplicationID. The more I think about it, the more it seems too complicated to keep track of. Shouldn't we just keep it simple and provide CSIProvisioner instead to identify the storage backend?

Reply:
Updated as a sub-struct as discussed. But left the targetID/provisioner in MMode CRD as is. Also changes it to StoraeProvisioner everywhere, instead of CSI term usage.